manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: FastStone.Viewer
+PackageVersion: "7.6"
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.faststonesoft.net/DN/FSViewerSetup76.exe
+  InstallerSha256: a19c82a4387ab1609a370027a78dce7528376d586a784af8e4aff42ab4c91e69
+ManifestType: installer
+ManifestVersion: 1.1.1
```

```suggestion
ManifestVersion: 1.1.0
```
`1.1.1` schema doesn't currently exist.